### ISSUE TITLE
Wrong ACC files

### ISSUE THREAD

**Issue by pbelmann** [metagenomics/bioPipeline#77]
*.acc files are wrong. E.g:

~~~shell
http://www.ncbi.nlm.nih.gov/protein/contig-5631000033_1
~~~

**Comment by pbelmann** [comment]
It seems that the *txt files produced by the blast step in our pipeline are created without extracting the protein id.

This is an example of an output of previous runs:

~~~shell
contig-2000039_211      gi|502758821|ref|WP_012993805.1|        38.64   220     125     6       7       222     2       215     1e-32     128   WP_012993805;YP_003474811;ADC90750      metallo-beta-lactamase domain-containing protein [Clostridiales genomosp. BVAB3]<>metallo-beta-lactamase domain-containing protein [Clostridiales genomosp. BVAB3 str. UPII9-5]<>metallo-beta-lactamase domain protein [Clostridiales genomosp. BVAB3 str. UPII9-5]    699245;699246 gi|502758821|ref|WP_012993805.1|;gi|289449778|ref|YP_003474811.1|;gi|289184325|gb|ADC90750.1|
~~~
As you can see, protein ids are extracted and listed (`WP_012993805;YP_003474811;ADC90750`)

But now a .txt file contains the following: 
~~~shell
contig-5631000033_1     gi|493241153|ref|WP_006212846.1|        25.95   289     186     6       62      349     84      345     3e-22   85.5    gi|493241153|ref|WP_006212846.1|        gi|493241153|ref|WP_006212846.1| D-alanyl-D-alanine carboxypeptidase [Paenibacillus vortex]    N/A     gi|493241153|ref|WP_006212846.1|
~~~

Instead of listing just the protein id  `WP_006212846.1`, further information is also included: `gi|493241153|ref|WP_006212846.1|` 

Benedikt can you check this?
(You can reproduce this with the data in [this](https://github.com/metagenomics/bioPipeline/tree/fix/acc-files) branch in the `features/data` directory)

**Comment by pbelmann** [comment]
The output depends on the input nr blast database.
